spotify-remote.el

```diff
@@ -53,16 +90,22 @@ See commands \\[spotify-toggle-repeating] and
   :group 'spotify
   :init-value nil
   :keymap spotify-mode-map
-  (let ((s '(:eval (spotify-mode-line-text))))
+  (let ((s '("    " (:eval (spotify-player-status-text)))))
```

Comment:
What is the reason for this particular number of spaces? Maybe turn this into a variable so that users can control this separator?